tekton/README.md

```diff
@@ -229,16 +246,69 @@ To run the `publish-tekton-pipelines` `Task` and create a release:
      -p "{\"secrets\": [{\"name\": \"$GENERIC_SECRET\"}]}"
    ```
 
-6. Run the `publish-tekton-pipelines` `Task`:
+4. To run the release you can either create a `PipelineRun` using
+   [`tkn`](https://github.com/tektoncd/cli), or using a yaml file.
 
-   ```bash
-   # If you are running in a cluster you've run this in previously, delete the previous run
+	You will need to set the following parameters:
+	- `versionTag`: to set the tag to use for published images
+	
+	  **TODO(#983) Be careful! if you use a tag that has already been released, you
+	  can overwrite a previous release!**
+	  
+	- `imageRegistry`: the default value points to
+      `gcr.io/tekton-releases`, to run against your own infrastructure
+      (not needed for actual releases) set it to your registry.
 
+6. Run the `release-pipeline`:
+
+   ```shell
+   # If you are running in a cluster you've run this in previously,
+   # delete the previous run and resources
+
+   # Apply golang tasks from the catalog
+   kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/golang/lint.yaml
+   kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/golang/build.yaml
+   kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/golang/tests.yaml
+
+   # Apply the publish Task
    kubectl apply -f tekton/publish.yaml
-   kubectl apply -f tekton/publish-run.yaml
 
-   # If you are running this in a shared cluster, delete the pipelinerun
+   # Create the resoruces
+   kubectl apply -f tekton/resources.yaml
+   ```
+
+   If you are using [`tkn`](https://github.com/tektoncd/cli), you can
+   run the following command.
+	
+   ```shell
+   # Do not forget to change those environment variables !
+   export VERSION_TAG=v0.X.Y
+   export SOURCE_REPO=tekton-pipelines-v0-X-Y
+
+   tkn pipeline start \
+		--param=versionTag=${VERSION_TAG} \
+		--serviceaccount=release-right-meow \
+		--resource=source-repo=${SOURCE_REPO} \
```

Comment:
i have some comments below - not sure it's worth making a new pipelineresource (manually) for each version?